Lib/uuid.py

```diff
@@ -214,7 +215,7 @@ def __init__(self, hex=None, bytes=None, bytes_le=None, fields=None,
             if not 0 <= int < 1<<128:
                 raise ValueError('int is out of range (need a 128-bit value)')
         if version is not None:
-            if not 1 <= version <= 5:
+            if not 1 <= version <= 7:
```

Comment:
is version 6 supported?

Reply:
Not yet but there is a PR for that. Note that version 2 is not implemented but we do not overcomplicate the check by omitting it.

Reply:
Also, strictly speaking, the UUID class is independent of whether the algorithm exists or not. You can technically use it for yourself, so it does not really matter I would say whether there is a factor for creating such version. You should just feed the constructor the correct values.